### ISSUE TITLE
SSDB in cloud

### ISSUE THREAD

**Issue by aigninja** [ideawu/ssdb#898]
Does anyone using ssdb in cloud like aws? As AWS EBS have restricted IOPS, How to get optimum output of ssdb on cloud?  Any pointer, case study, solution most welcome.

**Comment by ideawu** [comment]
Yes, Mobista uses ssdb in AWS. http://ssdb.io/docs/users.html#mobvista
